csv/src/test/scala/docs/scaladsl/CsvToMapSpec.scala

```diff
@@ -460,5 +460,5 @@ class CsvToMapSpec extends CsvSpec {
       )
     )
     // #header-line
-  }
+  }(mat)
```

Comment:
Where does this come from and where does it go?

Reply:
And it is required here: https://github.com/akka/akka/blob/5e9e11b5f6df03ed89c5ca37bdbb295ff5c19ec4/akka-stream-testkit/src/main/scala/akka/stream/testkit/scaladsl/StreamTestKit.scala#L28

Reply:
Hmm, but why isn't that available through implicit just for this last test case, it is for all the other test cases (via the actor system materializer) right?

Reply:
It was failing at some point, but now it seems to work when removing the explicit param again. Thanks for being nit-picky here!